cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,146 @@
+from django.test import TestCase, override_settings
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+
+
+class IndexPageTestCase(WagtailTestUtils, TestCase):
+    """Test IndexPage model properties and methods."""
+
+    def setUp(self):
+        self.index_page = IndexPageFactory(
```

Comment:
Suggestion: Create this page in `setUpTestData`, so it's not recreated with each case.

Reply:
Good shout! [2dfb6f4](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/2dfb6f4fb80bab1e5917fe81e27b1f1a55394914)